docs/services/vision/segmentation.md

```diff
@@ -209,28 +225,37 @@ To learn more about how to use segmentation, see the [Python SDK docs](https://p
 import (
 "go.viam.com/rdk/config"
 "go.viam.com/rdk/services/vision"
+"go.viam.com/rdk/components/camera"
 )
 
-visService, err := vision.FirstFromRobot(robot)
+// grab the camera from the robot
+cameraName := "cam1" // make sure to use the same component name that you have in your robot configuration
+myCam, err := camera.FromRobot(robot, cameraName)
 if err != nil {
-    logger.Fatalf("Cannot get Vision Service: %v", err)
+  logger.Fatalf("cannot get camera: %v", err)
 }
 
-segNames, err := visService.SegmenterNames(context.Background(), nil)
+visService, err := vision.from_robot(robot=robot, name='my_segmenter')
 if err != nil {
-    logger.Fatalf("Could not list detectors: %v", err)
+    logger.Fatalf("Cannot get Vision Service: %v", err)
 }
-logger.Info("Vision Resources:")
-logger.Info(segNames)
 
-// Apply the color segmenter to the image from your camera (configured as "camera_1")
-segments, err := visService.GetObjectPointClouds(context.Background(), "camera_1", "my_segmenter", nil)
+// gets the stream from a camera
+camStream, err := myCam.Stream(context.Background())
+
+// gets an image from the camera stream
+img, release, err := camStream.Next(context.Background())
+defer release()
+
+// Apply the color classifier to the image from your camera (configured as "cam1")
+segments, err := visService.GetObjectPointClouds(context.Background(), img)
```

Comment:
GetObjectPointclouds just needs a camera name -- this function form has to be a different different because it doesn't take in 2D images as an input